pages/learn/deploy/release-strategy/update-locking.md

```diff
@@ -17,10 +17,10 @@ On devices running supervisor 7.22.0 and higher, the lockfile is located at `/tm
 
 On older devices (with v4.0.0 <= supervisor version < v7.22.0) the lock is located at `/tmp/resin/resin-updates.lock`. The latest supervisor versions still take the lock at this legacy path for backwards compatibility.
 
-Legacy supervisors (< v4.0.0) have the lock at `/data/resin-updates.lock`. This lock is only supported on devices running resinOS 1.X.
+Legacy supervisors (< v4.0.0) have the lock at `/data/resin-updates.lock`. This lock is only supported on devices running balenaOS 1.X.
```

Comment:
perhaps here we should still refer to the 1.x line as resinOS? not really sure either way, since its all the same, but the wasn't really ever a balenaOS 1.x. Although this does get weird when we change it else where, so maybe its best to just call it balenaOS everwhere?

Reply:
I would prefer it if we could go balenaOS across the board but it's tricky as I guess the old versions are always going to report themselves as 'Resin OS' so it's going to need to be explained somewhere